src/os/CMakeLists.txt

```diff
@@ -115,16 +120,22 @@ endif()
 
 if(WITH_SPDK)
   include_directories("${CMAKE_SOURCE_DIR}/src/spdk/include")
+  add_custom_target(build_dpdk
```

Comment:
i think the first question would be: shall we force user to use the dpdk submodule instead of the one installed by the distro or by himself/herself?

@yuyuyu101 what do you think?